cogs5e/models/automation/effects/attack.py

```diff
@@ -235,6 +255,14 @@ def build_str(self, caster, evaluator):
             attack_bonus = stringify_intexpr(evaluator, self.bonus)
 
         out = f"Attack: {attack_bonus:+} to hit"
+        if self.adv:
+            match self.adv:
+                case 1:
+                    out += ", with advantage"
+                case 2:
+                    out += ", with Elven Accuracy"
+                case -1:
+                    out += ", with disdvantage"
```

Comment:
```suggestion
        match self.adv:
            case 1:
                out += ", with advantage"
            case 2:
                out += ", with Elven Accuracy"
            case -1:
                out += ", with disdvantage"
```
the `if` is redundant since patma will no-op if no cases are matched 😁